### PR TITLE
rdgo: Add ignition via fake distgit

### PR DESCRIPTION
**Pull request by ashcrow**

### CODE REVIEW

**Comment by cgwalters** [comment]
We need to delete the ignition COPR repo too right?

**Comment by ashcrow** [comment]
Yeah, we should. Doing so now.

File: overlay.yml [hunk 1]
```diff
@@ -21,5 +21,10 @@ components:
   - src: github:kubernetes-incubator/cri-o
     override-version: "1.11"
 
-#   - src: github:coreos/ignition
-#   TODO: use copr for distgit here OR get spec in the ignition repo
+  - src: github:coreos/ignition
+    override-version: "0.24.0"
```
Comment (jlebon) [inline overlay.yml]: We shouldn't need this, right? Looks like the tag is on the master branch. Would be good to keep those overrides to a minimum to keep maintenance low.

Comment (jlebon) [reply overlay.yml]:
```
$ git describe --tags --always --abbrev=42
v0.24.0-18-g042a00e218eb4a49eb22f053077c4e3152c8caed
```

**Comment by ashcrow** [comment]
Removed from the json file. Leaving the actual repo file there since it's still used in the OTA stuff for the moment. We'll rework that to take advantage of the stream later.

File: overlay.yml (hunk 1 above)
Comment (ashcrow) [reply overlay.yml]: I'll double check but you're likely correct.

Comment (ashcrow) [reply overlay.yml]: You are correct. I've removed it.

**Comment by jlebon** [comment]
Nice! /lgtm

**Comment by cgwalters** [comment]
@jlebon /lgtm needs to be on a new line I think

**Comment by jlebon** [comment]
Ahh, let me try that again.
/lgtm

**Pull request merged**